docs/getting-started.rst

```diff
@@ -1,4 +1,11 @@
 Getting Started
 ===============
 
+.. no CERN physicist? sorry, won't work
```

Comment:
add sth like "check out on how to use our open source project to build your own"?